### ISSUE TITLE
Add an 'All' filter

### ISSUE THREAD

**Issue by smichel17** [mpcjanssen/simpletask-android#594]
When we move from Saved Filters to Views, it'll be impossible not to have an active view. But when the user first launches the app, they haven't created any views yet, so we need a default one. It should be named `All`, should NOT have the option to update or delete it (although `Rename` and `To homescreen`  are fine), and should use the following settings:

#### OTHER

- [x] Show completed tasks
- [x] Show tasks with threshold date in the future
- [ ] Show hidden tasks
- [x] Show lists in the tasks
- [x] Show tags in the tasks
- [x] Show tasks' ages
- [ ] Create is threshold

#### SORT

\/ Alphabetical
\/ File Order (unsorted)

**Comment by ddisanti** [comment]
I would like to have an initial sorting as a default view, but it should be at user's choice. Right now, when the user applies a saved filter and then clears it, the list is left sorted as the last filter applied. Then you have to sort the list again. I think it was mentioned in another issue.

**Comment by smichel17** [comment]
tl;dr that's how it will work.

I have a vision that's bigger than just this issue, so many changes (like this one) won't make very much sense without context (and I haven't written everything out in any one place, so you're not going to be able to find the context easily).

In particular, this issue depends on 3 changes to make sense:
1. Switching to two-step filtering (giving the ability to clear the filter drawer without clearing the entire filter)
2. Switching the first step (saved filters, henceforth known as views) to be un-clearable; you always have an active view.
3. Allowing you to change the sorting from the main screen
  - Also, thinking about filtering as a different step from sorting. You won't be able to change the filter of the 'All' view because it'll always show *all* tasks; that's just what it is. But you will be able to change the sorting and do drawer-filtering on top of it.

We could wait and do everything in one go, but that is not really how @mpcjanssen and I are able to work, so we have to do it bit by bit, which leads to some weird scenarios like this issue, which doesn't make sense unless you have the info above for context.
